src/content/guides/asset-modules.mdx

```diff
@@ -78,6 +78,32 @@ module.exports = {
 }
 ```
 
+### Public Path
+
+By default, the `asset` type does `__webpack_public_path__ + import.meta` under the hood. This means that setting the `output.publicPath` in your config will allow you to override the url that the `asset` loads from.
+
+#### On The Fly Override
+
+If you set the `__webpack_public_path__` in code, the way you need to achive it so as not to break the `asset` loading logic is to make sure you run it as the first code in your app and not use a function to do so. An example of this would be having a file called `publicPath.js` with contents
```

Comment:
```suggestion
If you set the `__webpack_public_path__` in code, the way you need to achieve it so as not to break the `asset` loading logic is to make sure you run it as the first code in your app and not use a function to do so. An example of this would be having a file called `publicPath.js` with contents
```